python/pylibcugraph/pylibcugraph/experimental/__init__.py

```diff
@@ -76,16 +76,16 @@
 
 node2vec = promoted_experimental_warning_wrapper(node2vec)
```

Comment:
I know this isn't part of your PR, but can you remove this and the experimental code for it?  We should have removed this in an earlier release since the one-release grace period is over for it.  If this change is too big for this PR, can you file an issue instead?